### PR TITLE
:art: Add fade animation to cards

### PR DESCRIPTION
**Pull request by RicAlvesO**
If applied will add a new animation to selected/hovered cards in the cesium.link website.

### CODE REVIEW

File: components/Card/style.module.css [hunk 1]
```diff
@@ -11,14 +12,15 @@
   text-decoration: none;
   border: 1px solid #eaeaea;
   border-radius: 10px;
-  transition: color 0.15s ease, border-color 0.15s ease;
+  transition: color 0.5s ease, background-color 0.5s ease, border-color 0.5s ease;
```
Comment (nelsonmestevao) [inline components/Card/style.module.css]: These much transition time makes the website feel slow. Could you try a smaller value, please?

File: components/Card/style.module.css [hunk 2]
```diff
@@ -11,14 +12,15 @@
...
-.card:hover,
 .card:focus,
-.card:active {
-  color: blue;
-  border-color: blue;
+.card:active,
+.card:hover {
+  color: white;
+  border-color: white;
+  background-color: orange;
```
Comment (nelsonmestevao) [inline components/Card/style.module.css]: I like the upgrade to orange to make it more consist with our branding. However, I dislike the orange fill. Could you try orange as border color?

What about adding some animation effect on mouse hover, live rotating, shaking or something else?

**Review by nelsonmestevao — CHANGES_REQUESTED** [review]
Thank you for your first contribution :orange_heart: 

Could you check my comments, please? :pray:

**Comment by RicAlvesO** [comment]
Following the much appreciated feedback the button animation was changed. Transitions are back to .15s and background color is back to white.

As suggested, border and text color are now orange to be more in line with the brand, the new animation makes the orange color slide through the button as the color changes, disappearing right after.

**Review by nelsonmestevao — APPROVED** [review]
:rocket: 

Thanks again for this :pray:

**Pull request merged**